Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM openjdk:11
+ADD target/EmployeePortal-0.0.1-SNAPSHOT.jar employee-portal-app.jar
```

Comment:
How do you generate this `EmployeePortal-0.0.1-SNAPSHOT.jar` file?

Reply:
Generated the jar file using `mvn install -DskipTests`, Subbu.

Reply:
Got it. Thanks. Why can't that command be in Dockerfile?

Reply:
Every time the docker file is executed new jar file will be generated which is not necessary. So we can't have that command in Dockerfile, I guess.

Reply:
I can think of some drawbacks. Here they go
1. Generating the Jar(App build) in your local env rather than in docker is missing point of using the docker in the first place.
2. You local environment might have Java 8 and you docker environment must have Java 11. This might cause some issues/conflicts